src/utils/components/CPUMemoryModal/CpuMemoryModal.tsx

```diff
@@ -115,12 +119,13 @@ const CPUMemoryModal: React.FC<CPUMemoryModalProps> = ({ vm, isOpen, onClose, on
           key="default"
           variant={ButtonVariant.secondary}
           isDisabled={
+            !templateName ||
             !defaultsLoaded ||
             !templateDefaultsData?.defaultCpu ||
             !templateDefaultsData?.defaultMemory ||
             defaultLoadError
           }
-          isLoading={!defaultsLoaded}
+          isLoading={templateName && !defaultsLoaded}
```

Comment:
why do we need to add templateName to isLoading?

Reply:
Because why do we need to show the button loading if the button will be disabled anyway? It would be just misleading to the user.